pkg/commands/git_commands/branch.go

```diff
@@ -244,5 +246,17 @@ func (self *BranchCommands) Merge(branchName string, opts MergeOpts) error {
 }
 
 func (self *BranchCommands) AllBranchesLogCmdObj() oscommands.ICmdObj {
-	return self.cmd.New(str.ToArgv(self.UserConfig.Git.AllBranchesLogCmd)).DontLog()
+	// Only choose between non-empty, non-identical commands
+	candidates := lo.Uniq(lo.WithoutEmpty(append([]string{
+		self.UserConfig.Git.AllBranchesLogCmd,
```

Comment:
Since `AllBranchesLogCmd` has a default value, this means that if someone sets only the `AllBranchesLogCmds`, it will still prepend the default `AllBranchesLogCmd` to it.

So if someone wants to have only 2 custom log commands, they have to set `AllBranchesLogCmd` to empty as well, which isn't exactly deprecated then, is it?